app/src/main/res/layout/fragment_new_navigation.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#00ff00">
```

Comment:
This `xml` element can be closed here ----> `/>`.

Also, can we extract this `color`?